### PR TITLE
New chemical: Calcium. Adds a Calcium Oxide recipe.

### PR DESCRIPTION
**Pull request by Eneocho**
The Bone Juice(TM) is a dispensable chemical now and can be used to make Calcium Oxide (The heating reagent on ramen cups).
I plan on adding a few calcium-based afterwards.
:cl:
 * rscadd: New base chem: Calcium.
 * rscadd: You can make Calcium Oxide by mixing calcium, oxygen and pacid.
 * tweak: Calcium on the Chem Dispenser.
 * tweak: Degenerate Calcium now needs calcium instead of milk.

### CODE REVIEW

**Comment by Kurfursten** [comment]
This makes cream a lot more annoying to make in a small beaker like the one on the chef's table though. The chef does get a large in the grinder but it's often occupied and you don't want to leave 5u enzyme in it.

Why not just pick a recipe that doesn't require changing cream recipe? I do realize most people just pour out infinite cream at the bar nowadays but it seems odd. You could even make it a low-temperature reaction since it's something that heats up rapidly, I think that would be pretty cool.

You could also use degenerate calcium as a reagent, giving it a non-antag use as a process step. (degenerate calcium + oxygen?)

**Comment by Eneocho** [comment]
I would actually like for it to need elemental calcium, but we don't have that as a reagent, nor as an output of the chem dispensers nor as any ghetto source, so I just went with the milk one.
The idea was that you could use a chemical heat source to heat your reagents instead of the typical scalpel/bunsen burner methods. The degen calcium, while good in theory, makes it effectively purposeless, since you'd need to have a good heating source to make the degencalcium on the first place, though I guess it works as a placeholder chem until we have actual calcium as a reagent. 
Or I could just go with the previous iteration that needed pacid.

**Comment by Eneocho** [comment]
Should I make another PR for calcium as a chemdispenser reagent for atomicity or would it be ok to add it here?

**Comment by help-maint** [comment]
Is there more planned content for calcium, because adding a whole ass button to the chemdispenser for something that's used in 2 recipes is icky

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -1404,6 +1404,14 @@
 	if(prob(5))
 		M.emote(pick("twitch","drool","moan"), null, null, TRUE)
 
+/datum/reagent/calcium
+	name = "Calcium"
+	id = CALCIUM
+	description = "Calcium is an alkalyne earth metal, somewhat reactive when exposed to air. Good for the bones."
```
Comment (Exxion) [inline code/modules/reagents/Chemistry-Reagents.dm]:
```suggestion
	description = "Calcium is an alkaline earth metal, somewhat reactive when exposed to air. Good for the bones."
```

**Comment by Eneocho** [comment]
> 
> 
> Is there more planned content for calcium, because adding a whole ass button to the chemdispenser for something that's used in 2 recipes is icky

I do plan on adding a few reagents made with calcium, but they are not complete and adding them wouldn't be atomic. Besides, we already have a button for something that's only used on two recipes, namely: tungsten. Also mercury.

**Comment by Kurfursten** [comment]
I think it's clear that the new recipe for calcium oxide should be mercury + tungsten

**Comment by Eneocho** [comment]
Tungsten as catalyst because reasons.

**Comment by djkramer123** [comment]
mmm more useless chems on the chem dispenser

**Comment by GreatBigFailure** [comment]
Having calcium be a separate reagent does give me ideas for some easy to do recipes. Adding a whole button to the chem dispenser might be a bit much though.
What are some of your ideas for future reactions/effects? I think people would be a lot more receptive to it being in the dispenser if those were more outlined.

**Comment by Eneocho** [comment]
> 
> 
> What are some of your ideas for future reactions/effects? I think people would be a lot more receptive to it being in the dispenser if those were more outlined.

A tooth chem that fixes missing teeth made with mercury, gold and calcium (amalgam and bone juice) but poisons you while being metabolized, a chem opposite to ethylredox that turns water into ethanol made with calcium, tungsten and carbon, a chem opposite to calcium oxide that rapidly cools the holder upon mixing, a low explosive (either 1 tile breach and light damage or no breach and light to mid damage, haven't decided) made with mercury, carbon, nitrogen and oxygen with calcium as catalyst, a synthetic variant to anthracene that can glows in the dark if "charged" (you illuminate it, it stays bright until a certain time passes) made with calcium, oxygen and tungsten.

**Comment by Eneocho** [comment]
closing due to some stuff I wanna check that makes this outdated

**Comment by Kurfursten** [comment]
Calcium needs a recipe where you can cause the Queen Atom to leave the nest and cause a Helvetica Scenario

**Pull request closed without merging**